manifests/r/Ravencoin/RavenCore/4.6.1/Ravencoin.RavenCore.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.5-1-22621-608.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Ravencoin.RavenCore
+PackageVersion: 4.6.1
+InstallerLocale: en-US
+InstallerType: exe
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+ReleaseDate: 2022-08-19
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/RavenProject/Ravencoin/releases/download/v4.6.1/raven-4.6.1-7864c39c2-win64-setup-unsigned.exe
+  InstallerSha256: 828FAB42EAC2737D4BD431C67D5F6C2E4563C42B03A183CF9A31590C5602274F
```

```suggestion
  InstallerSha256: 828FAB42EAC2737D4BD431C67D5F6C2E4563C42B03A183CF9A31590C5602274F
  AppsAndFeaturesEntries:
  - DisplayName: Raven Core (64-bit)
    ProductCode: Raven Core (64-bit)
```
Additional metadata for better matching in upgrade scenario